Makefile

```diff
@@ -128,15 +134,15 @@ $(C_DIR_INCLUDE):
 $(C_LIBQISKIT): $(C_DIR_LIB)  $(C_LIB_CARGO_PATH)
 	cp $(C_LIB_CARGO_PATH) $(C_DIR_LIB)/$(subst _cext,,$(C_LIB_CARGO_FILENAME))
 
-$(C_QISKIT_H): $(C_DIR_INCLUDE) $(C_LIB_CARGO_PATH) 
+$(C_QISKIT_H): $(C_DIR_INCLUDE) $(C_LIB_CARGO_PATH)
 	cp target/qiskit.h $(C_DIR_INCLUDE)/qiskit.h
 
-.PHONY: c cheader 
+.PHONY: c cheader
 cheader: $(C_QISKIT_H)
 c: $(C_LIBQISKIT) $(C_QISKIT_H)
 
 # Use ctest to run C API tests
-ctest: $(C_LIB_CARGO_PATH) $(C_QISKIT_H) 
+ctest: $(C_LIB_DEBUG_CARGO_PATH) $(C_QISKIT_H)
```

Comment:
The tests are very small right now so compiling in Debug is fine, but wouldn't it generally make more sense to run in release mode for more speed? Or do you want to avoid building twice, once for coverage and once for the tests?

Reply:
I don't think it's really going to be an issue for a long time. Right now the compilation of the rust code and the test binary takes slightly more than 1.5min and the tests only take 0.02 sec. So we're a long time away from where I think we need to be worried about the test execution time. I generally think having the debug symbols for tests is a good thing too, although maybe not in CI.